src/main/java/imsam/Command.java

```diff
@@ -6,26 +6,36 @@
 public abstract class Command {
 
     @Option(name="-v",usage="enable verbose logging (INFO level)")
-    protected boolean verboseInfo = false;
+    protected boolean verboseLogging1 = false;
```

Comment:
Perhaps a better naming scheme could be `verboseLogLevelOne`, `verboseLogLevelTwo`, etc...